packages/react-events/src/__tests__/Focus-test.internal.js

```diff
@@ -351,6 +471,46 @@ describe('Focus event responder', () => {
         'inner: onFocusChange',
       ]);
     });
+
+    it('propagates focus within events', () => {
```

Comment:
events -> components

This isn't what we discussed so I'd like to understand why we should change event propagation for this component

Reply:
See previous comment about propagation change